### ISSUE TITLE
Accessibility improvements

### ISSUE THREAD

**Issue by Colin-Fredericks** [usablica/intro.js#426]
Hello! Thanks for creating Intro.js - I really appreciate it.

I made a few small alterations that improve the functionality for people who use screen readers. I wanted to suggest them to you here and see what you think.

After the line
`tooltipTextLayer.className = 'introjs-tooltiptext';`
add these three:

```
tooltipTextLayer.setAttribute('aria-live','assertive');
tooltipTextLayer.setAttribute('role','dialog');
tooltipTextLayer.setAttribute('tabindex','-1');
```

and instead of using `nextTooltipButton.focus();` or `skipTooltipButton.focus();`, which focus on the button (thus preventing VoiceOver from automatically speaking the description), instead focus the text. 

This makes VoiceOver automatically read the description, which is very beneficial. It's ugly because it highlights the text, but I added this bit of CSS to take that away:

```
.introjs-tooltiptext:focus {
  box-shadow:none;
  outline:none;
}
```

I'm sure this could be done better by someone who's an expert in accessibility; I just wanted to try to get a few steps closer to having this usable by blind and partially sighted folks, and I wanted to share it with you in case you found it useful.

Thanks again!

**Comment by mpgilbertusa** [comment]
I think this is a really great idea. The first three lines of setting attributes is a good start, but when I try to set focus on the introjs-tooltiptext box, it seems to break the whole thing. Did you have any luck getting it to work?

**Comment by Colin-Fredericks** [comment]
Thanks! I was using jQuery to do it (since it was available to me in the platform I'm using anyway), selecting the text by its class.

The specific lines I changed were 757, 760, and 932, all of which got replaced with 
`$('.introjs-tooltiptext').focus();`

I'm so used to jQuery that I don't remember the plain javascript equivalent. It looks like `getElementsByClassName` would work if you aren't worried about old versions of IE.

**Comment by mpgilbertusa** [comment]
That worked brilliantly! Thank you so much. :-)

**Comment by Colin-Fredericks** [comment]
Great! I'm glad to help.

Final note: I was able to test this in VoiceOver on a Mac, but I don't have a PC, so I'm not sure how JAWS will read it.

**Comment by mpgilbertusa** [comment]
One more addition I made to help with accessibility: I added some "visually hidden" text to the little gray circle links so that the screen reader could identify them properly. I added the following code after (the new) line 817...

    anchorLink.innerHTML = "<span class='visually-hidden'>step " + (this._introItems[i].step) + "</span>";

That way, the screen reader can speak "step 1", "step 2", etc. when you Tab to those gray links. It might also be nice to identify the current step, but I couldn't figure that out with the limited time I have to work on it today. Next time...
